src/plexosdb/db.py

```diff
@@ -2926,6 +2922,50 @@ def list_scenarios(self) -> list[str]:
         assert result
         return [d[0] for d in result]
 
+    def list_models(self) -> list[str]:
+        """Return all models in the database.
+
+        Returns
+        -------
+        list[str]
+            List of valid models names
+
+        Raises
+        ------
+        AssertionError
+            If the query fails
+
+        See Also
+        --------
+        query : Query the SQL database
+
+        Examples
+        --------
+        >>> db = PlexosDB()
+        >>> db.create_schema()
+        >>> db.add_object(ClassEnum.Generator, "Generator1")
+        >>> db.add_property(
+        ...     ClassEnum.Generator,
+        ...     "Generator1",
+        ...     "Max Capacity",
+        ...     100.0,
+        ...     model="Model",
+        ... )
+        >>> db.list_scenarios()
+        ["Model"]
```

Comment:
This docstring is wrong. Do something like `add_object(ClassEnum.Model, "model_01")` and then db.list_models()